selfdrive/locationd/laikad.py

```diff
@@ -203,6 +210,13 @@ def fetch_orbits(self, t: GPSTime, block):
           self.astro_dog.orbits, self.astro_dog.orbit_fetched_times, self.last_fetch_orbits_t = ret
           self.cache_ephemeris(t=t)
 
+  def kf_check_residual(self, residuals):
+    # if median residual is too large increase the Covariance matrix to improve convergence
+    if np.median(np.abs(residuals)) > RESIDUAL_THRESHOLD:
+      self.gnss_kf.init_state(self.gnss_kf.x, covs=self.gnss_kf.P_initial, filter_time=self.gnss_kf.filter.get_filter_time())
```

Comment:
"check" implies there are no side effects. rename function, or pull `init_state` out of the function.